docs/INSTALL.md

```diff
@@ -16,18 +16,12 @@ For _Windows_ you need to have [Git for Windows](https://git-scm.com/download/wi
 
 First of all get the source code. The full Organic Maps sources repository is ~8.5Gb in size, there are various [clone options](#special-cases-options) to reduce download size to suit your needs.
 
-For _Windows 10_ enable [symlinks](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks) support in git first:
```

Comment:
I think it's better to leave this part as is too. It worked for other contributors. And it is strange that it didn't work for you. Maybe it would make sense to debug it on your side, for example, by printing `git config --list`, or checking with some other test repos.

Also, the next line says: "Clone the repository...". If someone just clones it and then reads the next line on how to do it, it will fail, right?

Reply:
OK will try the git config --list and try to find some other repos. One person had the same trouble as me on discussion #3864, they do not say if they got it to work.